Announce ia and modes

brissaur opened the pull request.

Commit:
Design of playable cards and introducing announces

Commit:
Design of playable cards and introducing announces

Commit:
Design of playable cards and introducing announces

Commit:
Design of playable cards and introducing announces

Commit:
Remove general for top level fields && merge:true function

Commit:
Remove general for top level fields && merge:true function

Commit:
Remove general for top level fields && merge:true function

Commit:
Remove general for top level fields && merge:true function

Commit:
Make computer announce on announce and switch mode

- make IA player announce
- switch to play when 4 announces

Commit:
Make computer announce on announce and switch mode

- make IA player announce
- switch to play when 4 announces

Commit:
Make computer announce on announce and switch mode

- make IA player announce
- switch to play when 4 announces

Commit:
Make computer announce on announce and switch mode

- make IA player announce
- switch to play when 4 announces

xtrembaker commented on client/src/table/selectors.js:
```diff
@@ -21,22 +27,33 @@ export const getMyPlayer = createSelector(getPlayersRaw, getPlayerId, (players,
 
+export const getGameMode = createSelector(getGeneral, general => general.mode);
+
+export const isGameModeAnnounce = createSelector(getGameMode, gameMode => gameMode === 'announce');
```
Faudra que tu m'expliques comment ça marche, c'est pas clair pour moi :)

brissaur committed:
Possibility to announce (#23)

* Design of playable cards and introducing announces

* Remove general for top level fields && merge:true function

* Make computer announce on announce and switch mode

- make IA player announce
- switch to play when 4 announces

* WIP

- do remvoe announces
- error: https://console.firebase.google.com/u/1/project/card-game-coinche-ad92d/functions/logs?search=&functionFilter=onAnnounce&severity=DEBUG

Commit:
Resolving conflict

Commit:
Lint

Commit:
Lint

Commit:
Merge branch 'announce_ia_and_modes' of github.com:brissaur/card-game-coinche into announce_ia_and_modes

Commit:
Resolving conflict
